benchmarks/memcpy/memcpy.S

```diff
@@ -0,0 +1,111 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
I don't want to import GPL code into this repository.  Can you improve the `memcpy` in `syscalls.c`?  Unrolling the main loop, doing all the loads into local variables before doing the stores, should be enough to match the performance of this version, provided the input arrays are XLEN-bit aligned.

Reply:
Thanks, its now the unrolled syscalls.c version.